src/app/flows/fines/fines-acc/fines-acc-debtor-add-amend/constants/fines-acc-debtor-add-amend-field-errors.constant.ts

```diff
@@ -0,0 +1,278 @@
+      priority: 2,
+    },
+  },
+  facc_debtor_add_amend_contact_telephone_number_mobile: {
+    maxlength: {
+      message: 'Enter a valid mobile telephone number, like 07700 900 982',
+      priority: 1,
+    },
+    phoneNumberPattern: {
+      message: 'Enter a valid mobile telephone number, like 07700 900 982',
+      priority: 2,
+    },
+  },
+  facc_debtor_add_amend_contact_telephone_number_home: {
+    maxlength: {
+      message: 'Enter a valid home telephone number, like 01632 960 001',
+      priority: 1,
+    },
+    phoneNumberPattern: {
+      message: 'Enter a valid home telephone number, like 01632 960 001',
+      priority: 2,
+    },
+  },
+  facc_debtor_add_amend_contact_telephone_number_business: {
+    maxlength: {
+      message: 'Enter a valid work telephone number, like 01632 960 001 or 07700 900 982',
+      priority: 1,
+    },
+    phoneNumberPattern: {
+      message: 'Enter a valid work telephone number, like 01632 960 001 or 07700 900 982',
+      priority: 2,
+    },
+  },
+  facc_debtor_add_amend_vehicle_make: {
+    maxlength: {
+      message: `Make and model must be 30 characters or fewer`,
+      priority: 1,
+    },
+  },
+  facc_debtor_add_amend_vehicle_registration_mark: {
+    maxlength: {
+      message: `Registration number must be 20 characters or fewer`,
+      priority: 1,
+    },
+  },
+  facc_debtor_add_amend_language_preferences_document_language: {},
+  facc_debtor_add_amend_language_preferences_hearing_language: {},
+  facc_debtor_add_amend_employer_details_employer_company_name: {
+    maxlength: {
+      message: 'Employer name must be 50 characters or fewer',
+      priority: 2,
+    },
+  },
+  facc_debtor_add_amend_employer_details_employer_reference: {
+    maxlength: {
+      message: 'Employee reference must be 20 characters or fewer',
+      priority: 2,
+    },
+  },
+  facc_debtor_add_amend_employer_details_employer_email_address: {
+    maxlength: {
+      message: 'Employer email address must be 76 characters or fewer',
+      priority: 2,
+    },
+    emailPattern: {
+      message: 'Enter employer email address in the correct format, like name@example.com',
+      priority: 2,
+    },
+  },
+  facc_debtor_add_amend_employer_details_employer_telephone_number: {
+    maxlength: {
+      message: 'Enter a valid employer telephone number in the correct format, like 07700 900 982 or 01263 766122',
+      priority: 1,
+    },
+    phoneNumberPattern: {
+      message: 'Enter a valid employer telephone number in the correct format, like 07700 900 982 or 01263 766122',
+      priority: 2,
+    },
+  },
+  facc_debtor_add_amend_employer_details_employer_address_line_1: {
+    maxlength: {
+      message: 'Address line 1 must be 30 characters or fewer',
+      priority: 2,
+    },
+    alphanumericTextPattern: {
+      message: 'Address line 1 must only contain letters or numbers',
+      priority: 3,
+    },
```

Comment:
Incorrect priority number: 
```suggestion
    maxlength: {
      message: 'Address line 1 must be 30 characters or fewer',
      priority: 1,
    },
    alphanumericTextPattern: {
      message: 'Address line 1 must only contain letters or numbers',
      priority: 2,
    },
```